### DISCUSSION TITLE
Confirmed - v2 Working with HP Apollo 9000/735 PA-RISC Workstation

### DISCUSSION THREAD

**Discussion by papppmac** [erichelgeson/BlueSCSI-v1#224] [Show and tell]
I've used a BlueSCSIv2 connected to the narrow SCSI bus acting as both a hard disc and HP-UX 10.20 install CD to configure my HP Apollo 9000/735.  Absolutely no problems, I'd expect all the PA-RISC systems will be fine with it.

Initially with a HPDB50 to IDC50 adapter (the kind meant to go in a port cover, they're awesome SCSI tackle)  so I didn't have to screw with the cage, and later in place of the original hard drive on the ribbon.

**Comment by erichelgeson** [comment]
Thanks for the report - you can find v2's repo here though - https://github.com/BlueSCSI/BlueSCSI-v2/ - I've added it to the compatibility wiki there.

**Comment by pbertolo** [comment]
Having troubles with my 712. I think HD geometry is improperly set. Could you please share .ini file and a blank 2Gb image?

**Comment by gferluga** [comment]
Would work on a Visualize 8000?
What are the steps and configuration to perform?
Thx
